python/triton_kernels/triton_kernels/target_info.py

```diff
@@ -23,6 +25,15 @@
 ]
 
 
+@triton.constexpr_function
+def cuda_capability_geq(major, minor=0):
```

Comment:
what other properties are uncorrect for sm_120?

Reply:
Not sure what you mean by "other" or "incorrect properties". Without this workaround, the kernel tries to use native mxfp and TMA, assuming that sm120 has full features set of sm100. But those are the only things that are currently breaking gpt-oss on sm120 / 121.

Reply:
I meant in addition of the checks you modified. Do you know which use of `cuda_capability_geq` is causing problems